kernel/packages/shared/comms/index.ts

```diff
@@ -625,12 +626,13 @@ export async function connect(userId: string) {
         break
       }
       case 'v2': {
+        await ensureMetaConfigurationInitialized()
```

Comment:
@moliva Any objection to adding this method? Is there another way to await for meta config?

Reply:
I think it is ok. Metaconfiguration should be one of the first checkpoints setting the client up and should be up by this point. Being that we are not currently in a saga we have to use this kind of structures, but could later be changed to `waitForMetaConfigurationInitialization` (in `shared/meta/sagas` module) to avoid repetition.